website/docs/integrator-section/verifiable-document/ethereum/revoking-document.md

```diff
@@ -4,12 +4,37 @@ title: Revoking Documents
 sidebar_label: Revoking Documents
 ---
 
-After issuing a document, you might want to revoke it for any reason:
+Licenses, certifications, and permits are some examples of documents that can be issued as verifiable documents.
+
+When issuing verifiable documents, keep in mind that they will remain verifiable indefinitely.
+
+However, there may be times when the issued document needs to be revoked because: 
+
+* It contains errors.
+* It was issued wrongly.
+* After issuance, an incident occurs and requires the document to become ineffective and unverifiable.
+
+## Differences between revocation and expiration
+
+Revocation serves as a strict, proactive approach to prevent security breach or any change that may compromise the document integrity. 
+
+Document revocation is not intended to work as expiration, since documents with an expiry date already have a limited validity duration that the issuer determines at the time of file creation. 
```

Comment:
Can we highlight this in a colored box? something like 
![Screenshot 2023-12-15 at 17 16 05](https://github.com/Open-Attestation/documentation-website/assets/31716292/aad84c9e-f012-4c55-8df8-79402bbaecb0)

You can refer to https://docusaurus.io/docs/markdown-features/admonitions

Reply:
the text is highlighted in a blue box: 

<img width="1013" alt="info box" src="https://github.com/Open-Attestation/documentation-website/assets/93294344/cb785b48-7442-4cc8-9cb7-d1733bba6517">